lesson1/7findBiggestSquareURL.js

```diff
@@ -0,0 +1,33 @@
+const boxarts = [
+    {
+      width: 200,
+      height: 200,
+      url: "http://cdn-0.nflximg.com/images/2891/Fracture200.jpg"
+    }, 
+    {
+      width: 150,
+      height: 200,
+      url: "http://cdn-0.nflximg.com/images/2891/Fracture150.jpg"
+    }, 
+    {
+      width: 300,
+      height: 200,
+      url: "http://cdn-0.nflximg.com/images/2891/Fracture300.jpg"
+  }, 
+    {
+      width: 425,
+      height: 150,
+      url: "http://cdn-0.nflximg.com/images/2891/Fracture425.jpg"
+    }
+  ];
+
+boxarts.map(box => {
+    box.square = box.width * box.height
+})
+
+const biggestSquare = boxarts.reduce((acc, box) => {
+    acc.square < box.square ? acc = box : acc
+    return acc
+}, boxarts[0]).url
```

Comment:
here you can skip passing second argument to reduce. If second argument is not passed, reduce automatically takes first element at first iteration